eml/api/api.js

```diff
@@ -85,6 +121,78 @@ export const getAllComponents = async (components) => {
     components,
   };
   // Send request to S3 server
-  const res = await axios.post(url + '/api/component/getallcomponents', obj);
+  const res = await axios.post(url + '/api/components/all', obj);
```

Comment:
We should use singular words in URLs.

Reply:
i get error if i use singular words and then "all" afterwards...

Reply:
I assume this is due to the backend using 'components' too. Both of these should be fixed.

Reply:
I have change it in the backend as well, but the route that is called have been commented out and i am not sure why